gap/PolygonalComplexes/graphs.gd

```diff
@@ -812,6 +812,43 @@ DeclareAttribute( "EdgeGrapeGraph", IsPolygonalComplex );
 DeclareAttribute( "EdgeNautyGraph", IsPolygonalComplex );
 #! @EndGroup
 
+#! @BeginGroup
+#! @Description
+#! Return the face graph of a given polygonal surface. The vertices of the
+#! face graph are the faces of <A>surface</A> and for every edge in
+#! <A>surface</A> there is a corresponding edge in the face graph.
+#!
+#! The returned graph can be given in two different formats, corresponding
+#! to different graph packages:
+#! <K>Digraphs</K> and <K>NautyTracesInterface</K>
+#!
+#! The returned graph cannot be given as a grape graph because the <K>GRAPE</K> 
+#! package does not allow multiple edges.
+#! 
+#! Since the edges in <K>Digraphs</K> are directed but in the face graph they are not, 
+#! the method return for one undirected edge in the face graph two directed edges in the <K>Digraphs</K> package. 
```

Comment:
Sounds a bit strange. I suggest:
"Since the edges in <K>Digraphs</K> are directed but the face graph is undirected, each edge of the face graph is represented by two directed edges in the <K>Digraphs/K>-package.